references/README.md

```diff
@@ -2,17 +2,18 @@
 
 Welcome to the MeiliSearch API documentation.
 
-::: warning
+::: tip
 
-The documentation is written for the latest stable release: [v0.16.0](https://github.com/meilisearch/MeiliSearch/releases/tag/v0.15.0).
+You might find the answers to some of your questions [in the FAQ](/faq/faq.md) 💡
 
 :::
 
-::: tip
+## Version Compatibility
 
-You might find the answers to some of your questions [in the FAQ](/faq/faq.md) 💡
+The documentation of MeiliSearch is compatible with the latest version of MeiliSearch. We do not provide multi-versionning documentation.
+If you have a doubt concerning the version of [MeiliSearch](https://github.com/meilisearch/MeiliSearch), it is written on the upper navbar on the left side.
```

Comment:
"If you have a doubt about the version of Meilisearch" :  I find  "a doubt" very wide, I could have a doubt about the differences between the version I have installed and the latest one.  Or maybe I don't know which version I have installed in my computer. Maybe just change this to something like : "The actual version of Meilisearch is written..."